ci/ci.py

```diff
@@ -479,7 +479,7 @@ def main(argv):
                 return
 
         if platform and not is_platform_private(platform):
-            if platform not in ['x86_64-win32']:
+            if platform not in ['x86_64-win32', 'x86_64-linux']:
```

Comment:
For our defoldsdk.zip to build with linux tools (dlib_shared.so), we also need to build Linux